src/hb-ot-maxp-table.hh

```diff
@@ -39,6 +39,36 @@ namespace OT {
 
 #define HB_OT_TAG_maxp HB_TAG('m','a','x','p')
 
+struct maxpV1Tail
+{
+  inline bool sanitize (hb_sanitize_context_t *c) const
+  {
+    TRACE_SANITIZE (this);
+    return_trace (c->check_struct (this));
+  }
+
+  HBUINT16 maxPoints;             /* Maximum points in a non-composite glyph. */
+  HBUINT16 maxContours;           /* Maximum contours in a non-composite glyph. */
+  HBUINT16 maxCompositePoints;    /* Maximum points in a composite glyph. */
+  HBUINT16 maxCompositeContours;  /* Maximum contours in a composite glyph. */
+  HBUINT16 maxZones;              /* 1 if instructions do not use the twilight zone (Z0),
+                                     or 2 if instructions do use Z0; should be set to 2 in
+                                     most cases. */
+  HBUINT16 maxTwilightPoints;     /* Maximum points used in Z0. */
+  HBUINT16 maxStorage;            /* Number of Storage Area locations. */
+  HBUINT16 maxFunctionDefs;       /* Number of FDEFs, equal to the highest function number + 1. */
+  HBUINT16 maxInstructionDefs;    /* Number of IDEFs. */
+  HBUINT16 maxStackElements;      /* Maximum stack depth. (This includes Font and CVT
+                                     Programs, as well as the instructions for each glyph.) */
+  HBUINT16 maxSizeOfInstructions; /* Maximum byte count for glyph instructions. */
+  HBUINT16 maxComponentElements;  /* Maximum number of components referenced at
+                                     "top level" for any composite glyph. */
+  HBUINT16 maxComponentDepth;     /* Maximum levels of recursion; 1 for simple components. */
+
```

Comment:
Just by convention we always put the public: before the DEFINE_..., and with no empty line before it.

Reply:
Done.